NewPlatform.Flexberry.ServiceBus.Objects/Objects/Subscription.cs

```diff
@@ -147,6 +147,8 @@ public class Subscription : ICSSoft.STORMNET.DataObject, IDataObjectWithAuditFie
 
         // *** End programmer edit section *** (Subscription.Description CustomAttributes)
         [PropertyStorage("Описание")]
+        [DataServiceExpression(typeof(ICSSoft.STORMNET.Business.SQLDataService), "COALESCE(CAST(@Description@ AS VARCHAR), CONCAT(COALESCE(@Client.Name@, @Client.I" +
+            "D@), \' - \', COALESCE(@MessageType.Name@, @MessageType.ID@)))")]
```

Comment:
May be make 2 different DSE for MSSQL and PostgreSQL?

Reply:
What for?